homework-3/src/theme/index.js

```diff
@@ -6,4 +6,4 @@ const theme = {
   typography,
 };
 
-export default theme;
+export { theme };
```

Comment:
Why did you change export default?

Reply:
I don't remember why i changed this file
Fixed